### PR TITLE
Fix getting started

### PR DESCRIPTION
**Pull request by zkamvar**
this addresses #163

### CODE REVIEW

File: GETSTARTED.md [hunk 1]
```diff
@@ -2,10 +2,18 @@
...
+
+# Software
+
+- [Install R](http://cran.r-project.org/)
+- [Install RStudio](http://www.rstudio.com/)
+- [swirl: learn R interactively](http://swirlstats.com/)
+
+# Reference
+
+- [A few words for R beginners](https://www.biostars.org/p/123728/?utm_content=buffereadb1&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer)
```
Comment (hlapp) [inline GETSTARTED.md]: I think you want to remove the marketing/tracking parameters in the URL, i.e., everything after the `?` (and including the `?` itself).

**Comment by hlapp** [comment]
Just to double check, you did check through the menus and other docs that `LOAD_PACKAGES.md` isn't linked to from anywhere? I'm asking because I don't see removal of such links included in this PR.

File: GETSTARTED.md (hunk 1 above)
Comment (zkamvar) [inline GETSTARTED.md]: :+1:

**Comment by zkamvar** [comment]
> Just to double check, you did check through the menus and other docs that LOAD_PACKAGES.md isn't linked to from anywhere? I'm asking because I don't see removal of such links included in this PR.

Yes I did. LOAD_PACKAGES.md was one of the first files created that was never used because it was fairly redundant with information in other vignettes. 

All of these returned nothing:
```
grep  'LOAD_PACKAGES' *md
grep -inr 'LOAD_PACKAGES' use
grep -inr 'LOAD_PACKAGES' develop
grep -inr 'LOAD_PACKAGES' build
```

**Pull request merged**